myosuite/robot/robot.py

```diff
@@ -781,7 +782,7 @@ def reset(self,
 
     # close connection and exit out of the robot
     def close(self):
-        # prompt("Closing {}".format(self.name), 'white', 'on_grey', flush=True)
+        prompt("Closing {}".format(self.name), 'white', 'on_grey', flush=True)
```

Comment:
this throws errors in mac (M1) at closing

Reply:
Can you send the the command that throws the error

Reply:
I'll make another PR to manage this. Im resolving this

Reply:
```
Exception ignored in: <function Robot.__del__ at 0x13f667160>
Traceback (most recent call last):
  File "/Users/caggiano/Myo/myosuite/myosuite/robot/robot.py", line 792, in __del__
  File "/Users/caggiano/Myo/myosuite/myosuite/robot/robot.py", line 784, in close
TypeError: 'NoneType' object is not callable
```

Reply:
Somehow the enum Prompt is getting destroyed. I have made a workaround for now. It needs further investigation (in a different PR scope)